cli/batch/help.py

```diff
@@ -0,0 +1,8 @@
+"""
+Provide help messages for command line interface's batch commands.
+"""
+GET_BATCH_ID_ARGUMENT_HELP_MESSAGE = 'Get batch by identifier.'
+GET_BATCH_START_ARGUMENT_HELP_MESSAGE = 'Get batch by start.'
+GET_BATCH_LIMIT_ARGUMENT_HELP_MESSAGE = 'Limit amount of batches.'
+GET_BATCH_HEAD_ARGUMENT_HELP_MESSAGE = 'Get batch by head.'
+GET_BATCH_REVERSE_ARGUMENT_HELP_MESSAGE = 'Reverse result.'
```

Comment:
The same as here, you should describe the argument, not command. https://github.com/Remmeauth/remme-core-cli/pull/7/files#r276177736

Reply:
Done.